### ISSUE TITLE
nn.Linear are not support?

### ISSUE THREAD

**Issue by wangguojun2018** [NVIDIA-AI-IOT/torch2trt#124]
when the tensor that has `_trt` attribute are input `nn.Linear`, the output will loss the `_trt` attribute

**Comment by jaybdub** [comment]
Hi wangguojun2018,

Thanks for reaching out!  Linear is in fact supported, there was a bug when ``bias=False`` is set.

I've just fixed this bug in this pull request:  https://github.com/NVIDIA-AI-IOT/torch2trt/pull/125.  It is merged into master.

Please let me know if this works for you or if you have any issues.

Best,
John

**Comment by wangguojun2018** [comment]
I  clone the newest master, ran reinstall by `python setup.py install`, but it not make sense. The `_trt` still is lost by `nn.Linear(bias=False)`

**Comment by wangguojun2018** [comment]
And I try `nn.Linear(bias=True)`, it was also lost the `_trt`

**Comment by jaybdub** [comment]
Hi,

Are you able to share the model that you're using?

Best,
John

**Comment by wangguojun2018** [comment]
> Hi,
> 
> Are you able to share the model that you're using?
> 
> Best,
> John

Ok,this is my source code. `my_model` and `PillarFeatureNet` both have the error `'Tensor' object has no attribute '_trt' `, I think  the `PillarFeatureNet` error was caused by the operations like this `points_mean = features[:, :, :3].sum(
            dim=1, keepdim=True) / num_voxels.type_as(features).view(-1, 1, 1)
        f_cluster = features[:, :, :3] - points_mean`. However ,the error of `my model ` was caused by `nn.Linear`. The attachment is my source code. Thank you for your reply!
[torch2trt_test.txt](https://github.com/NVIDIA-AI-IOT/torch2trt/files/3753644/torch2trt_test.txt)

**Comment by xieydd** [comment]
same error @wangguojun2018 have you solve the error?
